bin/analyzedata.js

```diff
@@ -0,0 +1,270 @@
+#!/usr/bin/env node --harmony
+
+var program = require('commander');
```

Comment:
Ideally new files should be written in ES6. We can convert existing files if/when we have time to do so, but new files should be ES6, and confirm to the airbnb eslint standard (see https://github.com/tidepool-org/chrome-uploader/blob/pazaan/medtronic-600/lib/drivers/medtronic600/.eslintrc)